src/envoy/mixer/http_control.cc

```diff
@@ -124,6 +125,12 @@ void FillRequestInfoAttributes(const AccessLog::RequestInfo& info,
     attr->attributes[kAttrNameResponseTime] =
         Int64Value(info.duration().count());
   }
+  if (info.responseCode().valid()) {
+    attr->attributes[kStatusCode] =
+        StringValue(std::to_string(info.responseCode().value()));
+  } else {
+    attr->attributes[kStatusCode] = StringValue("500");
```

Comment:
Not sure if responseCode() already covers ones rejected by Envoy, such as rate limiting.

If no, we may need to check getResponseFlag.